src/components/badge/Badge.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const prefix = 'tk-badge';
+
+export type BadgeProps = {
```

Comment:
can you extend div props? (you can check other components that already do that ;)

Reply:
done